roles/strongswan/tasks/openssl.yml

```diff
@@ -155,10 +155,25 @@
         format: OpenSSH
       with_items: "{{ users }}"
 
+    - name: Get OpenSSL version
+    shell: |
```

Comment:
@vvorlov could you fix the indent please?

- https://github.com/trailofbits/algo/actions/runs/8962593883/job/24704944056?pr=14718#step:6:1435